RELEASE.md

```diff
@@ -1,3 +1,25 @@
+# Release 1.3.0
+
+## New features
+
+- Dockerfile optimisation (https://github.com/marqo-ai/marqo/pull/569/files). A pre-built Marqo base image results in reduced image layers and increased build speed.
+- New E5 models added to model registry (https://github.com/marqo-ai/marqo/pull/568). E5 V2 and Multilingual E5 models are now available for use. See all available models [here] (https://marqo.pages.dev/1.2.0/Models-Reference/dense_retrieval/).
```

Comment:
Move this above the Dockerfile optimisation, as it has the most impact on most open source Marqo users